docs/cloud/satellites/gha-runners.md

```diff
@@ -0,0 +1,168 @@
+# Satellites as GHA runners (**experimental**)
+
+Earthly Satellites embed now a GitHub self-hosted runner, so they can directly pull jobs from GHA without the need of an intermediate runner.
+
+This self-hosted runner comes with the Earthly CLI preinstalled, and it's configured to use the Satellite BuildKit instance, so GHA jobs will share the same Satellite cache than the traditional Satellite builds.
+
+Also, notice that the self-hosted runner can run any arbitrary GHA job, not necessarily an Earthly command, and given that it runs within the Satellite, it benefits from its persistent local storage (see ["Persistent Folders" section below](#persistent-folders)) across builds.
```

Comment:
```suggestion
The self-hosted runner can run _any_ GHA job, not necessarily an Earthly command. Since it runs on the Satellite, it benefits from its persistent local storage (see ["Persistent Folders" section below](#persistent-folders)) across builds.
```